passes/src/Luna/Pass/Resolution/MethodResolution.hs

```diff
@@ -56,12 +57,16 @@ runMethodResolution :: MonadPassManager m => SubPass AccessorFunction m Bool
 runMethodResolution = do
     accs <- getAccs <$> getAttr @UnresolvedAccs
     res <- forM accs $ \acc -> do
-        result <- importAccessor acc
         case result of
-            False -> return $ Just acc
-            _     -> return Nothing
-    putAttr @UnresolvedAccs $ putAccs $ catMaybes res
-    return $ any isNothing res
+            True -> return $ Just acc
```

Comment:
Where does `result` come from? Does the code compile?

Reply:
2 lines above, WDYM?

Reply:
Didn't know you can push amended commits and github doesn't mind :(

Reply:
It's a feature, and a pretty good one I'd say.